build/packer/filebeat/filebeat-agent.yml

```diff
@@ -0,0 +1,28 @@
+filebeat.inputs:
```

Comment:
will sync file from [roachsiem](https://github.com/cockroachlabs/roachsiem/) repo.

Reply:
Might be worth getting @jlinder  to look at this pattern too since he owns this process at the moment.

Reply:
My bad on the terminology side, by 'syncing' I meant we will have to manually update this file whenever there is a change to the file in the roachsiem repo, since it's private and isn't easily accessible on the image.

Reply:
Ah. Ok. How often do you think that will change? (hoping it's not to often because the process for updating this is a pain in the butt).

Reply:
Shouldn't be too often, at most whenever there's a change to the way artifacts are released or the logs associated with them. 

This is all of the logic we wanted to monitor build logs for now.

Reply:
Can you add a comment in the file for what needs to be updated when it changes on the roachsiem side? and if there is code on the roachsiem side, point to this file in comments once this file is merged?

Reply:
Added comments.